addons/amb_civ_command/fnc_cc_rogueTarget.sqf

```diff
@@ -63,7 +63,7 @@ switch (_state) do {
         //Thank you, BIS...
         if (_targetSide in ["GUER","INDEP"]) then {_targetSide = RESISTANCE} else {
             if (_targetSide in ["CIV","CIVILIAN"]) then {_targetSide = CIVILIAN} else {
-                _targetSide = call compile _targetSide;
+                _targetSide = missionNamespace getVariable [_targetSide,WEST];
```

Comment:
this might be west/east which are not variables but commands. I'd say just add a extra check for them.

Reply:
@marceldev89 can you shed some light here?

Reply:
@dedmen is correct. We're passing around string versions of whatever side it is and further down the pipeline we compare it with `side player` (or whatever object) so it needs to be compiled down to a `Side` type.